.changeset/yellow-wombats-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```

Comment:
this should be admin.applications.v1, not console

Reply:
Fixed with 6b02ab6bdf7323798ec96467cf93e3ac68dd4690